Bukkit/build.gradle.kts

```diff
@@ -69,7 +72,6 @@ bukkit {
 tasks {
     withType<ShadowJar> {
         dependencies {
-            include(dependency("org.mcstats.bukkit:metrics-lite"))
         }
```

Comment:
Is that dependency part even needed when it doesn't have anything in it?

Reply:
We didn't wanna touch a lot of what was there prior